src/components/org/apache/jmeter/visualizers/backend/influxdb/InfluxdbBackendListenerClient.java

```diff
@@ -318,6 +335,13 @@ public void setupTest(BackendListenerContext context) throws Exception {
                 }
             }
         }
+        // Check if more fields are filled ( corresponding to user tag )
+        context.getParameterNamesIterator().forEachRemaining(name->{
+            if ( !defaultArg.containsKey(name)) {
+                TAG_USER += "," + AbstractInfluxdbMetricsSender.tagToStringValue(name.trim()) + "=" + AbstractInfluxdbMetricsSender.tagToStringValue(context.getParameter(name).trim());
```

Comment:
Thanks for this PR. It would require more documentation as I find the usage a bit edgy. AFAIU you 'll enable user to add as many rows (key=value) which will be tags for Influx event.
Can't we have some meaningful prefix or something to avoid a conflict between a user tag and a real property of the InfluxdbBackendListenerClient ?

Reply:
I don't think the user will add a lot of tag and it's more clear for me in this way and it's easier to parse data into influxdb line protocol. 
If you want, we can add a '_' at the start of each user tag but for me it's not mandatory as real property of the InfluxdbBackendListenerClient are default setted

Reply:
What if user names his tag the same as an existing attribute ? he can't . If we add a prefix "TAG_" then I feel it's more clear and avoid conflicts. 
Also I think this needs more documentation. Feature without documentation is not usable. We currently lack documentation on InfluxDBBackendListenerClient.
I think it's a feature that is very interesting and we need users to know it exists and what it does.